src/util/options.rs

```diff
@@ -189,7 +189,8 @@ options! {
     // Perf events to measure
     // Semicolons are used to separate events
     // Each event is in the format of event_name,pid,cpu (see man perf_event_open for what pid and cpu mean)
-    perf_events:           PerfEventOptions     [always_valid] = PerfEventOptions {events: vec![]}
+    work_perf_events:       PerfEventOptions     [always_valid] = PerfEventOptions {events: vec![]},
```

Comment:
Is it necessary to allow reading different perf events for work and for phase? It would be more straightforward to juse the same set of events for both. Please add comments for each option.

Reply:
If we use the same set of events, it might create too many counters (2x the number of events, one for process-wide and one for work packets) than what the CPU can support. So sometimes you might want more counters for the process-wide measurement than for work packets or vice versa.

Reply:
You can consider using `work_perf_events` and `phase_perf_events` to differentiate them.